pkg/plugin/probes-handler/probes_handler_test.go

```diff
@@ -0,0 +1,64 @@
+package probeshandler_test
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"os"
+
+	probeshandler "github.com/arquillian/ike-prow-plugins/pkg/plugin/probes-handler"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Test liveliness and readiness probes.", func() {
+
+	const (
+		probesEndpoint = "/version"
+		version        = "xxxxxxxx-xxxxxxxxxx"
+	)
+
+	var _ = BeforeSuite(func() {
+		os.Setenv("VERSION", version)
+	})
+
+	var _ = AfterSuite(func() {
+		os.Clearenv()
+	})
+
+	Context("When in healthy state", func() {
+		It("should return plugin version in response body", func() {
+			// given
+			probesHandler := probeshandler.NewProbesHandler()
+			request := httptest.NewRequest("GET", probesEndpoint, nil)
+			response := httptest.NewRecorder()
+			expectedBody := probeshandler.Probe{Version: version}
+
+			// when
+			http.Handle(probesEndpoint, probesHandler)
+			probesHandler.ServeHTTP(response, request)
+
+			// then
+			actualBody := probeshandler.Probe{}
+			json.Unmarshal(response.Body.Bytes(), &actualBody)
+			Expect(actualBody).To(Equal(expectedBody))
+		})
+	})
+
+	Context("When in unhealthy state", func() {
+		It("should return empty response body", func() {
```

Comment:
btw, does anything from this test verify anything from the probe functionality?

Reply:
My intention with this test was to test for the unhealthy state. The test does not verify the probe functionality directly but what I tried testing here was the end-point.

Reply:
Removing the test as discussed offline.